pandas/tests/test_generic.py

```diff
@@ -1539,6 +1540,15 @@ def test_to_xarray(self):
                            expected,
                            check_index_type=False)
 
+    def test_deepcopy_empty(self):
```

Comment:
look at how the other tests are done. We need systematic testing on all structures. So move to ``Generic`` and use ``_construct`` and ``_compare``